ogticrd/covid-safe-paths
# Fix: Change unknown in idioms

#### Description:


I changed the unknown label in idioms and put it the label depends on the language of the app

#### Linked issues:


Fixes [#92](https://trello.com/c/VeI3amf5/92-change-unknown-en-el-idioma)
#### Screenshots:
![image](https://user-images.githubusercontent.com/43874147/83659536-ab3d8c80-a591-11ea-8453-c005d6def17e.png)



#### How to test:


- `git pull`
- `git checkout Fix/92-idioms`
- Head over to settings screen and change the language
- it should change the label and the language of settings
